packages/rs-drive-abci/src/rpc/core.rs

```diff
@@ -105,35 +135,37 @@ impl CoreRPCLike for DefaultCoreRPC {
     }
 
     fn get_transaction(&self, tx_id: &Txid) -> Result<Transaction, Error> {
-        self.inner.get_raw_transaction(tx_id, None)
+        retry!(self.inner.get_raw_transaction(tx_id, None))
```

Comment:
Is it retry for timeout error only? I don't see how retry could help in other cases. If Core is not responding then it won't isn't it?